app/assets/stylesheets/fonticon_picker.scss

```
@@ -1,17 +1,55 @@
 .fonticon-picker-modal {
-  height: calc(100vh - 260px);
-  overflow-y: auto !important;
-  overflow-x: hidden;
-
+  
   div.fonticon {
     font-size: 24px;
     text-align: center;
+    padding: 0;
 
-    span.active {
-      i {
+    span { 
+      i{
```

```suggestion
      i {
```
or make line 19 like this one - whichever style is right